osidb/models.py

```diff
@@ -1130,13 +1135,17 @@ def _validate_unsupported_impact_change(self, **kwargs):
         ] or old_flaw.major_incident_state in [
             Flaw.FlawMajorIncident.APPROVED,
             Flaw.FlawMajorIncident.CISA_APPROVED,
+            # Minor Incident is only informative
+            Flaw.FlawMajorIncident.ZERO_DAY,
```

Comment:
Maybe it is only me, but the way the Minor incident is commented kinda feels like the whole comment regards to `ZERO_DAY` instead and it creates confusion that Zero day is Minor. I got actually confused during the review. Maybe I would include the whole name in the comment `Flaw.FlawMajorIncident.MINOR is only informative`. This repeats several times.

Reply:
I will think about how to make it less confusing. I just wanted to explicitly mention that the minor one is not included on purpose and not just forgotten etc.

Reply:
I reworded it and used your advice